src/components/nav.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { MdPerson } from 'react-icons/md';
+
+const Nav = () => (
+  <nav className="nav">
+    <div className="container">
+      <h1>BookStore CMS</h1>
+      <ul className="menu">
+        <Link to="/">
+          <li className="navlink" title="home">
+            BOOKS
+          </li>
+        </Link>
+        <Link to="/calculator">
+          <li className="navlink" title="calculator">
+            CATEGORIES
+          </li>
+        </Link>
```

Comment:
- [ ] For this page to be rendered correctly, I think the path should be `**categories**` as stated in your routes path `(app.js)`. At the moment, nothing is displayed when I clicked on CATEGORIES link. 👍 

<img width="265" alt="Untitled" src="https://github.com/Mikey1-nuel/Bookstore/assets/103386183/80165044-1d31-45cf-8c6d-8285885c2bc8">